iternity-dotcom/minio
# Use continuation marker properly

## Description

Previously the continuation marker was not evaluated correctly.

## Motivation and Context

This slightly speeds up subsequent list requests.

## How to test this PR?

The easiest way to verify this is to run a performance comparison between iternity-rb and these changes.

## Types of changes

## Checklist:
